compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/config_user_vector_distribute.mlir

```diff
@@ -81,15 +81,15 @@ hal.executable public @main_0_dispatch_0 {
 
 // -----
 
-// Check that applying the `reorder_workgroups = transpose` unit attribute enables workgroup reordering.
+// Check that applying the `ReorderWorkgroupsTranspose` pipeline option attribute enables workgroup reordering.
 
 // OPT-OUT:       #[[$TRANSLATION:.+]] = #iree_codegen.translation_info<LLVMGPUVectorDistribute workgroup_size = [128, 2, 1] subgroup_size = 64
+// OPT-OUT-SAME:    gpu_pipeline_options = #iree_gpu<gpu_pipeline_options[ReorderWorkgroupsTranspose]>
 // OPT-OUT-SAME:    mma_schedule = #iree_gpu.mma_schedule<intrinsic = #iree_gpu.mma_layout<MFMA_F32_16x16x16_F16>,
-// OPT-OUT-SAME:    reorder_workgroups = "transpose"
 
 // OPT-IN:       #[[$TRANSLATION:.+]] = #iree_codegen.translation_info<LLVMGPUVectorDistribute workgroup_size = [128, 2, 1] subgroup_size = 64
+// OPT-IN-SAME:    gpu_pipeline_options = #iree_gpu<gpu_pipeline_options[ReorderWorkgroupsTranspose]>
```

Comment:
What happens when there are multiple conflicting options specified? Is this reported as an error in any way?

Reply:
Options are part of structured attr parameters now, so this is no longer an issue.